CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.5.1 (January 21, 2021)
+
+ BUG FIXES:
+
+ * Moved subnet validation from customizediff to create SDDC block [\#96](https://github.com/vmware/terraform-provider-vmc/pull/96)
+ * Added zerocloud check for setting intranet MTU uplink [\#98](https://github.com/vmware/terraform-provider-vmc/pull/98)
+ * Updated help documentation for import SDDC functionality [\#97](https://github.com/vmware/terraform-provider-vmc/pull/97)
```

Comment:
We have updated the documentation with the limitations for import. Wouldn't it be better to state it explicitly?

Reply:
done